app/models/display_ad.rb

```diff
@@ -27,7 +27,7 @@ def process_markdown
     markdown = Redcarpet::Markdown.new(renderer)
     initial_html = markdown.render(body_markdown)
     # Temporarily disable the sanitisation in order to launch the SheCoded Campaign.
-    # TODO: [@thepracticaldev/cool] find an alternate solution.
+    # TODO: [@forem/cool] find an alternate solution.
```

Comment:
@forem/cool no longer exists, who should this be assigned to?

Reply:
I think no one. I would remove the parenthesis in the `TODO`